matcher_test.go

```diff
@@ -126,16 +174,35 @@ func TestMatcher(t *testing.T) {
 	      ["field", "isActive"],
 	      ["value", true]
 	    ]
-	  ],
```

Comment:
I'm disabling "anyin" for now. Will re-enable it once SimpleParser has it implemented.